students/bec_lukasz/lesson_03_functions/exponentiation.py

```diff
@@ -0,0 +1,9 @@
+
+def power(a, n):
+    return a * a ** (n - 1)
```

Comment:
Statement
Given a positive real number a and a non-negative integer n.
Calculate an without using loops, ** operator or the built in
function math.pow(). Instead, use recursion and the relation
an=a⋅an−1. Print the result.
Form the function power(a, n).

Użyłeś `**`, gdzie powinna być zastosowana rekursja :).